node/src/PipeTransport.ts

```diff
@@ -1,4 +1,5 @@
-import { v4 as uuidv4 } from 'uuid';
+import { randomUUID } from 'node:crypto';
```

Comment:
I know that importing from `node:xxxx` instead of from `xxxx` is nicer nowadays but we cannot change this just here and I'd appreciate if PRs don't change cosmetic opinionated styles despite they are valid.

Reply:
I see that `node:xxxx` is added in other places but I don't want unrelated unneeded style changes in a PR.